README.md

```diff
@@ -6,6 +6,15 @@ Express.JS frameworks.
 
 ## Features
 
+The features below are there for two basic reasons. First reason is to showcase
+some of the features of the Knative (like scaling, traffic shaping, or event
+sending and receiving). The other group is to showcase some basic features
+almost every production ready application deployed on Kubernetes should have.
+The non-functional features were added, so the apps be close to the real world 
+apps, and could serve as a reference for developers.
+
+### Knative related features
```

Comment:
Hmm, I thought it would be obvious. Especially for contributors like you. If it isn't, then we need to work on the README to make it clear.

Are the explanation in the other comment: https://github.com/openshift-knative/showcase/pull/11#issuecomment-1505877259 make it more clear?

Reply:
I will comment on https://github.com/openshift-knative/showcase/pull/11#issuecomment-1505877259

Reply:
Commented [here](https://github.com/openshift-knative/showcase/pull/11#discussion_r1165133413)